Black all the things

j-bennet opened the pull request:
Running `black .` shouldn't make unintended changes. We can use `pyproject.toml` to keep `black` rules.

pquentin reviewed (changes requested):
Thanks for fixing this! However, we are now storing the list of included folders in two places. What are your thoughts on just running black/isort on everything, including changelog.py and docs/conf.py?

This would fix `black .`, remove some extra config, and allow me to edit changelog.py and docs/conf.py with my editor without ruining the formatting.

j-bennet commented:
After running `black` on everything, 3 files ended up with changes:

```
	modified:   changelog.py
	modified:   docs/conf.py
	modified:   docs/create_scheduler_plot.py
```

If there's no specific reason to include `docs`, I'd rather `black` everything.

pquentin commented on docs/conf.py:
```diff
@@ -30,17 +30,15 @@
-extensions = [
-    "sphinx.ext.ifconfig"
-]
+extensions = ["sphinx.ext.ifconfig"]
```
```suggestion
extensions = ["sphinx.ext.ifconfig",]
```

pquentin commented on docs/conf.py:
```diff
@@ -51,14 +49,10 @@ def read_min_python_version():
-GLOBAL_SUBSTITUTIONS = {
-    "{MIN_PY_VER}": read_min_python_version()
-}
+GLOBAL_SUBSTITUTIONS = {"{MIN_PY_VER}": read_min_python_version()}
```
```suggestion
GLOBAL_SUBSTITUTIONS = {"{MIN_PY_VER}": read_min_python_version(),}
```

pquentin commented on docs/conf.py:
```diff
@@ -132,62 +126,57 @@ def read_min_es_version():
 latex_documents = [
-    (master_doc, 'Rally.tex', 'Rally Documentation',
-     'Daniel Mitterdorfer', 'manual'),
+    (master_doc, "Rally.tex", "Rally Documentation", "Daniel Mitterdorfer", "manual"),
```
```suggestion
    (master_doc, "Rally.tex", "Rally Documentation", "Daniel Mitterdorfer", "manual",),
```

pquentin commented on docs/conf.py:
```diff
@@ -132,62 +126,57 @@ def read_min_es_version():
-    (master_doc, 'esrally', 'Rally Documentation',
-     [author], 1)
-]
+man_pages = [(master_doc, "esrally", "Rally Documentation", [author], 1)]
```
```suggestion
man_pages = [(master_doc, "esrally", "Rally Documentation", [author], 1,)]
```

pquentin commented on docs/conf.py:
```diff
@@ -132,62 +126,57 @@ def read_min_es_version():
-    (master_doc, 'Rally', 'Rally Documentation',
-     author, 'Rally', 'Macrobenchmarking framework for Elasticsearch.',
-     'Miscellaneous'),
+    (master_doc, "Rally", "Rally Documentation", author, "Rally", "Macrobenchmarking framework for Elasticsearch.", "Miscellaneous"),
```
```suggestion
    (master_doc, "Rally", "Rally Documentation", author, "Rally", "Macrobenchmarking framework for Elasticsearch.", "Miscellaneous",),
```

pquentin reviewed (commented):
Thank you! Yes, your new proposal is the right thing to do: there is no good reason why those were left out, they were not linted so I did not format them.

I left a few trailing commas as suggestions, this will tell black to restore those lines as blocks.

pquentin reviewed (approved):
Thanks for bearing with me! LGTM.
